scripts/check_version_match.py

```diff
@@ -15,4 +15,4 @@
         f"Version in pyproject.toml ({version_in_pyproject_toml}) does not match version in "
         f"cognite/client/_version ({__version__})"
     )
-    exit(1)
+    raise SystemExit(1)
```

Comment:
Is this the latest n' greatest in exit methods? 😁

Reply:
Haha yes! I learned it from the Python-Youtube-God himself ( @asottile ) 😂